src/tests/ftest/launch.py

```diff
@@ -27,14 +27,48 @@
 import json
 from multiprocessing import Process
 import os
+import re
 import socket
 import subprocess
 import time
 import yaml
 
+try:
+    # For python versions >= 3.2
+    from tempfile import TemporaryDirectory
+
+except ImportError:
+    # Basic implementation of TemporaryDirectory for python versions < 3.2
+    from tempfile import mkdtemp
+    from shutil import rmtree
+
+    class TemporaryDirectory(object):
+        # pylint: disable=too-few-public-methods
+        """Create a temporary directory.
+
+        When the last reference of this object goes out of scope the directory
+        and its contents are removed.
+        """
+
+        def __init__(self):
+            """Initialize a TemporaryDirectory object."""
+            self.name = mkdtemp()
+
+        def __del__(self):
+            """Destroy a TemporaryDirectory object."""
+            rmtree(self.name)
+
 
 TEST_LOG_FILE_YAML = "./data/daos_avocado_test.yaml"
 BASE_LOG_FILE_YAML = "./data/daos_server_baseline.yaml"
+SERVER_KEYS = (
+    "test_machines",
+    "test_servers",
+    "daos_servers",
+    )
+CLIENT_KEYS = (
```

Comment:
Now that this is a single key, can it be removed?

Reply:
I don't think it should be removed.  It should remain a constant as the value is referenced in three separate methods.  I could see the definition being changed from a list to a string to avoid the perception of additional keys being an option.  This could be addressed - along with doing the same to `SERVER_KEYS` - in the PR that addresses the multiple server keys.

Reply:
Fair enough.